Tools/Scripts/webkitpy/port/factory.py

```diff
@@ -150,7 +150,10 @@ def get(self, port_name=None, options=None, **kwargs):
         for cls in classes:
             if port_name.startswith(cls.port_name):
                 port_name = cls.determine_full_port_name(self._host, options, port_name)
-                return cls(self._host, port_name, options=options, **kwargs)
+                try:
+                    return cls(self._host, port_name, options=options, **kwargs)
+                except ValueError:
```

Comment:
When do we expect this to raise? Do we want to log anything when this happens?

Reply:
We don't want to log anything, the exception being raised on line `157` is the correct error to raise, in this case. We expect this to happen when someone requests an invalid port, like `visionos-wk1`, for example.